.vscode/launch.json

```diff
@@ -17,7 +17,7 @@
             "name": "Debug Debugger",
             "type": "node",
             "request": "launch",
-            "program": "${workspaceRoot}/out/debugger/nodeDebugWrapper.js",
+            "program": "${workspaceRoot}/src/debugger/reactNativeDebugEntryPoint.ts",
```

Comment:
Yep, it works. I was using this launch task to debug `nodeDebugWrapper`. Also, the official documentation on node debugging suggests that `"program"` points to *.js file, but I've seen a lot of examples where *.ts file is used directly as a program to launch.